### ISSUE TITLE
Дока + Dash

### ISSUE THREAD

**Issue by solarrust** [Y-Doka/y-doka.site#363]

**Comment by pepelsbey** [comment]
[Dash for macOS](https://kapeli.com/dash)

**Comment by Realetive** [comment]
Да, есть [Velocity](https://velocity.silverlakesoftware.com/) — порт для Windows, совместимый с оригиналом. Но я тоже не считаю эту задачу приоритетной до релиза.

**Comment by igor-arkhipov** [comment]
Dash может использовать и репозиторий в качестве доксета. Алсо, несколько dash docset generator существуют
